CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
     - `keyType`: keys of the map
     - `valueType`: values of the map
 - Added support for `include_nulls` argument in `DataFrame.unpivot`.
+- Added client-side thin-client mode which can be enabled with `session.ast_enabled=True` (default: `False`).
```

Comment:
enable ast does not enable a thin client mode right? We still send sql query to the server.

Reply:
change to
`Added parameter `ast_enabled` to session for internal usage (default: False)`